winforms/tests_backend/widgets/base.py

```diff
@@ -61,6 +56,16 @@ def assert_font_family(self, expected):
             SYSTEM: SystemFonts.DefaultFont.FontFamily.Name,
         }.get(expected, expected)
 
+    def assert_font_options(self, weight, style, variant):
+        if style == OBLIQUE:
+            print("Ignoring OBLIQUE font test")
+        elif SMALL_CAPS:
+            print("Ignoring SMALL CAPS font test")
```

Comment:
Shouldn't OBLIQUE fall back on ITALIC?

Also, even if SMALL_CAPS isn't supported, the weight and style should still be tested.

Reply:
I'm not sure I'm convinced that OBLIQUE should be considered a substitute for ITALIC. Substitutions are always going to be a bit sketchy - I'd rather make the substitution that involves the least stylistic judgement on our part :-)

Agreed that we should be testing every property that isn't in a fallback state, though.

Reply:
[According to MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/font-style#values), CSS does fall back from oblique to italic, and vice versa.

Reply:
Huh.. TIL. Updated the implementation to reflect this.